docusaurus/docs/tools/user_guide/poktrolld_cli.md

```diff
@@ -43,19 +42,14 @@ poktrolld version
 poktrolld --help
 ```
 
-:::tip
-See the [homebrew-poktroll](https://github.com/pokt-network/homebrew-poktroll/)
-repository for details on how to install homebrew or other details to install
-or debug the CLI.
-:::
-
-### Troubleshooting Homebrew
-
-:::info Having issues with Homebrew?
-
+<details>
```

Comment:
Moved troubleshooting information behind a spoiler, so the user can see the next installation option.